Assets/GothicVR/Scripts/Player/BasicMovement/ControllerManager.cs

```diff
@@ -47,8 +46,6 @@ private void Update()
             //Use the first configuration set of controller buttons
             ControllerManager_v1();
```

Comment:
Please remove the _v1 suffix. It's the final version.

Reply:
Changed for now.

The _v1 was not meant as version 1. It was a preparation to one possible solution of button configuration. 
The idea was to have 2 or 3 predefined sets of buttons. A user could pick between the 3 options. That was meant to be easier to implement and handle for us than to make each button individually configurable.

But I changed it for now, until we have a way forward on the controls.